AK/JsonArray.h

```diff
@@ -27,8 +28,8 @@ class JsonArray {
     {
     }
 
-    template<typename T>
-    JsonArray(Vector<T> const& vector)
+    template<IterableContainer ContainerT>
+    JsonArray(ContainerT const& vector)
```

Comment:
`vector` should probably be renamed since it won't always be a Vector now. Maybe `source`?